cache/redisOps.js

```diff
@@ -142,15 +142,16 @@ function deleteKey(type, name) {
  * @param  {String} type - The type of the master list on which the
  *  set operations are to be performed
  * @param  {String} objectName - The object name (like Subject, Aspect, Sample)
- * @param {String} name - Name of the key to be deleted
+ * @param {String} _name - The name of the key to be deleted
```

Comment:
prefer leave the arg name in the fn signature just as "name" (not prefixing with underscore char)